worlds/meritous/Rules.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2022 FelicitusNeko
+#
+# This software is released under the MIT License.
+# https://opensource.org/licenses/MIT
+
+import typing
+from ..generic.Rules import add_rule
+from .Regions import meritous_regions, connect_regions
+
+
+def check_endgame(state, player):
+    for check in ["Meridian", "Ataraxia", "Merodach"]:
+        if not state.can_reach(check, "Location", player):
```

Comment:
I think this should be "Region" rather then "Location"

Reply:
Scrap that, i finely had time to test it. and it makes it wurse

Reply:
Thank you for doing so on my behalf. I just had my COVID booster shot yesterday¸ and have little energy for much right now.

I feel like, when I just marked everything as a progression item, it resulted in more successes. However, I have to figure that this generally shouldn't be how one does things in this engine.

Also, I think I need to revise the completion objective; currently, if I understand correctly, it'll signal as complete if the Cursed Seal is accessible, rather than having collected it and went in to beat the final boss. Maybe that's why it marks as unbeatable? Some of the progression items are locked behind the end condition? But then wouldn't it fail every time?

Reply:
Nope. Tried adding a virtual item for victory and linking to that as win condition, and still no luck. I have no idea what I'm doing wrong. :/

The code for this comment is outdated, so I'm going to resolve this conversation, but the problem still exists.